tests/private-rbac-validation.unit.test.ts

```
@@ -0,0 +1,98 @@
+  createRoleBindingBodyHook,
+  createRoleBindingBodySchema,
+  invalidOrgIdHook,
+  orgIdParamSchema,
+  updateGroupBodyHook,
+  updateGroupBodySchema,
+  updateRoleBindingBodyHook,
+  updateRoleBindingBodySchema,
+} from '../supabase/functions/_backend/private/rbac_validation.ts'
+
+type ValidationIssues = readonly StandardSchemaV1.Issue[]
+type StandardSchema = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(ts/consistent-type-definitions)